hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/SparkBaseIndexSupport.scala

```diff
@@ -78,7 +78,7 @@ abstract class SparkBaseIndexSupport(spark: SparkSession,
   def invalidateCaches(): Unit
 
   def getPrunedPartitionsAndFileNames(prunedPartitionsAndFileSlices: Seq[(Option[BaseHoodieTableFileIndex.PartitionPath], Seq[FileSlice])],
-                                      includeLogFiles: Boolean = false): (Set[String], Set[String]) = {
+                                      includeLogFiles: Boolean = true): (Set[String], Set[String]) = {
```

Comment:
Should we pass the `includeLogFiles` in HoodieFileIndex?
https://github.com/apache/hudi/blob/ae5833ea34afc8849722509d185f4e94dd3fb424/hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieFileIndex.scala#L93